src/pages/leaderboard.js

```diff
@@ -108,6 +108,7 @@ const Leaderboard = ({ data }) => {
         gasOptz: 0,
         allFindings: 0,
         awardTotal: 0,
+        rank:0
```

Comment:
Usually not so compact, and with trailing commas (when supported)

```suggestion
        rank: 0,
```